src/EA.Weee.Web/RazorHelpers/WeeeGds.Warning.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace EA.Weee.Web.RazorHelpers
+{
+    using System.Web.Mvc;
+
+    public partial class WeeeGds<TModel>
```

Comment:
if you have time, would you add a test for this helper

Reply:
added...